packages/@react-spectrum/picker/stories/Picker.stories.tsx

```diff
@@ -551,3 +564,150 @@ function ResizePicker() {
     </Flex>
   );
 }
+
+// eslint-disable-next-line rulesdir/sort-imports
+import {DismissButton, useOverlay} from '@react-aria/overlays';
+import {FocusScope} from '@react-aria/focus';
+import {HiddenSelect, useSelect} from '@react-aria/select';
+import {mergeProps} from '@react-aria/utils';
+import {useButton} from '@react-aria/button';
+import {useFocus} from '@react-aria/interactions';
+import {useListBox, useOption} from '@react-aria/listbox';
+import {useSelectState} from '@react-stately/select';
+
+function Select(props) {
+  // Create state based on the incoming props
```

Comment:
Example is taken from docs and is just for testing purposes, will remove if this approach is suitable